.gitattributes

```diff
@@ -3,3 +3,4 @@
 *.descriptor_set  linguist-generated=true
 *.descriptor_set  -diff -merge
 *.pb.html linguist-generated=true
+*.pb.go linguist-generated=true
```

Comment:
/lgtm

We made a similar change to istio/api [here](https://github.com/istio/api/pull/741/files/0be6db77a77a098d72cc24d6ec89af734ac350de#diff-fc723d30b02a4cca7a534518111c1a66).